geode-core/src/main/java/org/apache/geode/internal/cache/CacheDistributionAdvisor.java

```diff
@@ -142,17 +149,25 @@ public String toString() {
    *
    * @param excludeInRecovery if true then members in recovery are excluded
```

Comment:
Update this comment by removing the @param and saying something like "members that are currently in recovery will not be in the returned set.